core/src/main/java/org/frankframework/senders/IbisLocalSender.java

```diff
@@ -113,7 +114,7 @@
 @Category(Category.Type.BASIC)
 public class IbisLocalSender extends AbstractSenderWithParameters implements HasPhysicalDestination, IThreadCreator, ConfigurationAware {
 
-	private final @Getter String domain = "Local";
+	private final @Getter DestinationType domain = DestinationType.LOCAL;
```

Comment:
Ik vind de destinations `LOCAL` en `JVM` niet erg duidelijk. Misschien moeten we er `PIPELINE` van maken voor beiden? Of `ADAPTER`?
Dat maakt het duidelijk dat een andere pipeline aanroept.

Alleen als je via de DLL een aanroep doet naar iets anders dan is het `JVM`, niet altijd `PIPELINE`.